### PR TITLE
conductor.md updated to reflect v1alpha2 api changes

### LINKED ISSUE #251: conductor: update spec doc to reflect new api changes
blocked by #205 #207

### PR DESCRIPTION
**Pull request by sambukowski**
## Summary
The [v1alpha2 api](https://buf.build/astria/astria/docs/main:astria.execution.v1alpha2#astria.execution.v1alpha2.ExecutionService.GetBlock) is currently being implemented. This pr updates conductor.md to reflect those changes.

## Testing
The changes in this pr are only to markdown documents. No testing required.

## Related Issues
#205 #207

closes #251

### CODE REVIEW

File: specs/conductor.md [hunk 1]
```diff
@@ -48,37 +52,28 @@ Conductor is written in Rust and utilizes the Tokio runtime to achieve this.
...
-
-The Driver receives either `Events`
-([link](https://github.com/astriaorg/astria/blob/6e71a76fa52c522ffdcabcd9d659e4de765d9d61/crates/astria-gossipnet/src/network_stream.rs#L39))
-from the network, or `DriverCommand`s
-([link](https://github.com/astriaorg/astria/blob/6e71a76fa52c522ffdcabcd9d659e4de765d9d61/crates/astria-conductor/src/driver.rs#L54))
+- Creates a `TendermintClient` to validate Sequencer blocks received from the
+  sequencer
+-  Passes validate sequencer blocks to the Executor for execution on the rollup
+
+The Driver receives either `Events` ([link](https://github.com/astriaorg/astria/blob/6e71a76fa52c522ffdcabcd9d659e4de765d9d61/crates/astria-gossipnet/src/network_stream.rs#L39))
```
Comment (noot) [inline specs/conductor.md:63]: remove `Events` (it's part of gossipnet)

File: specs/conductor.md [hunk 2]
```diff
@@ -48,37 +52,28 @@ Conductor is written in Rust and utilizes the Tokio runtime to achieve this.
...
+from the network, or `DriverCommand`s ([link](https://github.com/astriaorg/astria/blob/6e71a76fa52c522ffdcabcd9d659e4de765d9d61/crates/astria-conductor/src/driver.rs#L54))
 from the Conductor's internal event loop on a timer. The variants for `Event`
```
Comment (noot) [inline specs/conductor.md:65]: remove `Event`

File: specs/conductor.md [hunk 3]
```diff
@@ -48,37 +52,28 @@ Conductor is written in Rust and utilizes the Tokio runtime to achieve this.
...
 and `DriverCommand` that are relevant to the processing of blocks within the
 Conductor are:
 
-- `Event::GossipsubMessage(Message)`
-  ([link](https://github.com/astriaorg/astria/blob/6e71a76fa52c522ffdcabcd9d659e4de765d9d61/crates/astria-gossipnet/src/network_stream.rs#L50))
+- `Event::GossipsubMessage(Message)` ([link](https://github.com/astriaorg/astria/blob/6e71a76fa52c522ffdcabcd9d659e4de765d9d61/crates/astria-gossipnet/src/network_stream.rs#L50))
```
Comment (noot) [inline specs/conductor.md:69]: remove?

File: specs/conductor.md [hunk 4]
```diff
@@ -152,36 +149,34 @@ The `chain_id` that Conductor uses as the rollup's identifier is pulled from the
...
+api](https://github.com/astriaorg/astria/blob/main/specs/execution-api.md) for more details.
 
 ### Data Validation
 
 Data is validated before being sent to the rollup for execution. Validation
 occurs in two places:
 
-- When blocks are received from the gossip network, the data is validated in
+- When blocks are received directly from the sequencer, the data is validated in the
   Driver's `handle_network_event` using
```
Comment (noot) [inline specs/conductor.md:163]: this probably won't be called `handle_network_event` after

File: specs/conductor.md [hunk 5]
```diff
@@ -152,36 +149,34 @@ The `chain_id` that Conductor uses as the rollup's identifier is pulled from the
...
   `BlockVerifier::validate_rollup_data`
 
 ### Soft Commitments
 
-When a Sequencer block is received by the Conductor from the gossip network, it
-is assumed to be the `head` of the chain, thus its parent block is designated as
-a `soft` commitment. This is also the primary moment in the data life cycle for
-Astria that transactions are sent to the rollup for execution.
+When a block is received by the Conductor directly from the sequencer, it
+is assumed to be valid based on CometBTF consensus and immediately set as a `soft` commitment.
```
Comment (noot) [inline specs/conductor.md:172]:
```suggestion
is assumed to be valid based on CometBFT consensus and immediately set as a `soft` commitment.
```

Comment (noot) [reply specs/conductor.md:172]: also it isn't assumed to be valid, we should fetch the commit for it + check that as well (has not been implemented)

File: specs/conductor.md [hunk 6]
```diff
@@ -152,36 +140,33 @@ The `chain_id` that Conductor uses as the rollup's identifier is pulled from the
 rollup's config. When a user submits a transaction to be sequenced, they specify
 the `chain_id` of its destination.
 
-When a Sequencer block is received from the gossip network, the Conductor
+When a Sequencer block is received, the Conductor
```
Comment (noot) [inline specs/conductor.md:143]: received from websocket

**Review by noot — APPROVED** [review]
nice!

**Review by joroshiba — APPROVED** [review]

**Pull request merged**